### ISSUE TITLE
ChocolateyGUI stuck on "Retrieving installed packages"

### ISSUE THREAD

**Issue by esamo** [chocolatey/ChocolateyGUI#266]
Fresh Win8 x64 system, `chocolateygui` worked for a while - two programs installed, then `notepad++` and after gui restart it claims to be `retrieving installed packages`, but no resources or hdd is under load.

---

Regards & respect.

**Comment by esamo** [comment]
Okay after few system restarts, GUI works as expected again, no longer stuck on start.

**Comment by RichiCoder1** [comment]
Let us know if it pops up again.

**Comment by suimong** [comment]
I got stuck at the same place with only chocolatey and chocolateyGUI installed. Windows 8.1 x64 system. Got around it by setting the compatibility level to Windows 7, but still think it is a bug to fix.
Version info:
chocolatey: 0.9.9.6
chocolateyGUI: 0.13.1

**Comment by Ytrog** [comment]
I have the same on Win 7 with GUI 0.13.2 and choco 0.9.9.11

**Comment by dkirrane** [comment]
I still have this problem on Windows 7 Professional 64-bit (6.1, Build 7601)
ChocolateyGUI version 0.13.2.0

**Comment by DauphinCountyLibrarySystem** [comment]
Just a heads-up, I experienced this same issue with Chocolatey 0.9.9.11 and ChocolateyGUI 0.13.2.

As per esamo's experience, I rebooted twice and the issue seems to have resolved itself.

**Comment by Silberling** [comment]
First installation of chocolatey. First package was ChocolateyGUI. Getting "Retrieving installed packages...", too. It's like this for about an hour now.

**Comment by illwieckz** [comment]
Exact same issue there, ChocolateyGUI was the first package installed by Chocolatey.

Almost first… Chocolatey automatically installed PowerShell 5 as a dependency, then returned to prompt and displayed a warning saying a reboot is needed to finish the PS5 install.

I ignored that warning, tried to install ChocolateyGUI but it failed as expected. I rebooted. Windows said some stuff about some update installing (I guess PS5 finishing to install), then I retried to install rebooted ChocolateyGUI, it worked, but I got it stuck at fist startup. I just quit it (no reboot), then restarted and it was OK.

Same Chocolatey and ChocolateyGUI version as @Silberling, on Windows 7 64bit.

**Comment by RichiCoder1** [comment]
Should be fixed w/ latest develop 0.15.

**Comment by maphew** [comment]
I got the same problem with new Win7 x64 install, nothing but Chcoloatey, Powershell and Ch-GUI installed. Rebooting did not help.

```
C:\Windows\system32>choco list -l
Chocolatey v0.10.3
chocolatey 0.10.3
ChocolateyGUI 0.13.2
PowerShell 5.1.14409.20170301
3 packages installed.
```

What did work was upgrading to pre-release v0.14.0 with (from project readme.md):

    choco upgrade chocolateygui --source https://www.myget.org/F/chocolateygui/ --pre
